src/OSPSuite.SimModel/include/SimModel/FormulaChange.h

```diff
@@ -22,7 +23,8 @@ struct formulaParameterInfo {
 };
 
 class FormulaChange : 
-	public XMLLoader
+	public XMLLoader,
+	public EntityWithCachedScaleFactor
```

Comment:
FormulaChanes caches factor of species (in case the corresponding changed entity is a species)

Reply:
FromulaChange describes change of 1 entity (Parameter, ODE-Variable) by event.
E.g if an event is defined by
```
If Time==10 then 
  P1=P1+1;   
  Y1=0;
end
```
Then 2 formulaChanges will be defined: one changes parameter P1, one changes variable Y1